modules/rc_fax/composer.json

```diff
@@ -0,0 +1,7 @@
+{
+    "autoload": {
+        "psr-4": {
+            "RingCentral\\SDK\\": "vendor/ringcentral/ringcentral-php/src"
```

Comment:
you should be able to fold this into the main composer.json

Reply:
This api installed by composer pulls in 5 libraries not needed or currently used by openemr(no choice). I want this api static and composer is there for class loader update only, not api install...

Reply:
Is the offender the pubnub/pubnub package?
https://packagist.org/packages/ringcentral/ringcentral-php
(there is a cool thing we can do in composer.json to ignore a dependency package if this is the case)

Reply:
I don't mind however ya want.